pkgs/development/ocaml-modules/gd4o/default.nix

```diff
@@ -12,6 +12,9 @@ stdenv.mkDerivation rec {
   buildInputs = [ ocaml findlib libjpeg libpng ];
   propagatedBuildInputs = [ gd zlib freetype ];
 
+  prePatch = ''
+    substituteInPlace Makefile --replace gcc cc
+  '';
```

Comment:
```suggestion
  
  makeFlags = [
    "CC=${stdenv.cc.targetPrefix}cc"
  ];
```

Reply:
Weird I thought command-line variable have lower precedence than variable defined with `CC=gcc` in makefile and we must use `CC ?= gcc` to allow such override.